doc/nrf/ug_bt_ll_nrfxlib.rst

```diff
@@ -0,0 +1,33 @@
+.. _ug_bt_ll_nrfxlib:
+
+nRF BLE Controller (experimental)
+################################################
+
+This user guide describes the integration of the nRF BLE Controller with the Zephyr RTOS.
+The nRF BLE Controller described in this document should not be confused with the open source BLE Controller included in the Zephyr project.
+The nRF BLE Controller is distributed as a set of pre-compiled linkable libraries which can be found in the :ref:`nrfxlib` repository (???).
```

Comment:
@ru-fu, @b-gent how would you like to link to the linkable libraries? E.g.:
https://github.com/thomasstenersen/nrfxlib/tree/ble-controller-re-revert/ble_controller/lib/cortex-m4/hard-float